google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -761,4 +762,31 @@ def do_rollback(self, dbapi_connection):
         ):
             pass
         else:
-            dbapi_connection.rollback()
+            trace_attributes = {"db.connection": dbapi_connection}
```

Comment:
`dbapi_connection` is an object which means the trace attribute is going to show a value like: 
```
"dbapi_connection": <google.cloud.spanner_dbapi.connection.Connection object at 0x7fd1b01e3880>
```

Instead, set the `db.instance` attribute with `dbapi_connection.database.name` so that the span will have the URI of the database, e.g.
```
"db.instance": "projects/<project>/instances/<instance>/databases/<database>"
```